packages/common/src/Components/FileInput.jsx

```diff
@@ -40,11 +40,12 @@ class FileInput extends Component {
           {...a11yProps}
           className={styles.visuallyHidden}
           id={this.id}
-          type={'file'}
+          type={(!onClick) ? 'file' : ''}
           data-hook={dataHook} onChange={onChange}
           accept={accept}
           onFocus={() => this.onFocus()}
           onBlur={() => this.onBlur()}
+          onClick={onClick || null}
```

Comment:
if `onClick` is null or undefined you can just leave `onClick`
```suggestion
          onClick={onClick}
```